bin/concert

```diff
@@ -395,6 +398,15 @@ class StartCommand(Command):
         # Add session path, so that sessions can import other sessions
         sys.path.append(cs.path())
 
+        # ctrl-c handling
+        orig_sigint_handler = signal.getsignal(signal.SIGINT)
```

Comment:
@matze the time I spent googling this is not so long, so do you perhaps know a cleaner way of doing this?

Reply:
I think this is alright. Signals are weird anyway, so …